packages/r3f/src/components/SnapshotEditor.tsx

```diff
@@ -50,9 +51,11 @@ const EditorScene: React.FC<{snapshotEditorSheet: ISheet; paneId: string}> = ({
   const showGrid = useVal(editorObject?.props.viewport.showGrid) ?? true
   const showAxes = useVal(editorObject?.props.viewport.showAxes) ?? true
 
+  const grid = useMemo(() => new InfiniteGridHelper(), [])
+
   return (
     <>
-      {showGrid && <gridHelper args={[20, 20, '#6e6e6e', '#4a4b4b']} />}
+      {showGrid && <primitive object={grid} />}
```

Comment:
I think you could use extend here

```
extend({ InfiniteGridHelper })

...
<infiniteGridHelper />
```

Reply:
Oh yeah it was complaining it was not part of the THREE namespace and I was too lazy to do anything about it